easybuild/easyconfigs/o/OVITO/OVITO-2.9.0-GCCcore-6.4.0-x86_64.eb

```diff
@@ -0,0 +1,29 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - http://hprc.tamu.edu
+
+easyblock = 'PackedBinary'
+
+name = 'OVITO'
+version = '2.9.0'
+versionsuffix = '-x86_64'
+
+homepage = 'https://www.ovito.org/'
+
+description = """
+ OVITO is a scientific visualization and analysis software
+ for atomistic simulation data
+"""
+
+source_urls = [('https://www.ovito.org/download/%(version)s')]
+sources = ['%(namelower)s-%(version)s%(versionsuffix)s.tar.gz']
+checksums = ['2eb484483dfe97003d0c1885899abdeb3dea00499a7a186b2f8811aa91d62052']
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
```

Comment:
It needs a newer libstdc++ than what RHEL6 currently provides.   I just went with the latests GCCcore (I think GCC 5.x might work as well).

Should GCCcore just be a dependency instead of the toolchain?